modules/oe_content_reference_code_field/config/install/field.storage.node.oe_reference_codes.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
```

Comment:
We need an update path for this configuration.

Reply:
This is done.